actors/builtin/miner/testing.go

```diff
@@ -491,20 +502,18 @@ func CheckExpirationQueue(expQ ExpirationQueue, liveSectors map[abi.SectorNumber
 			} else {
 				acc.Addf("on-time expiration sector %d isn't live", n)
 			}
-
 			return nil
-		}); err != nil {
-			return err
-		}
+		})
+		acc.RequireNoError(err, "error iterating on-time sectors")
 
-		if err := exp.EarlySectors.ForEach(func(n uint64) error {
+		err = exp.EarlySectors.ForEach(func(n uint64) error {
 			sno := abi.SectorNumber(n)
 			// Check sectors are present only once.
 			acc.Require(!seenSectors[sno], "sector %d in expiration queue twice", sno)
 			seenSectors[sno] = true
 
 			// Check early sectors are faulty
-			acc.Require(partitionFaultsMap[n], "sector %d expiring early but not faulty", sno)
+			acc.Require(partitionFaultsMap != nil && partitionFaultsMap[n], "sector %d expiring early but not faulty", sno)
```

Comment:
I'd prefer `partitionFaultsMap == nil || partitionFaultsMap[n]` here. If we can't load the partition faults map, this could generate lots of redundant messages.